src/audio/tdfb/tdfb.c

```diff
@@ -281,6 +401,11 @@ static struct comp_dev *tdfb_new(const struct comp_driver *drv,
 	cd->fir_delay = NULL;
 	cd->fir_delay_size = 0;
 
+	/* Defaults for enum controls */
+	cd->az_value = 0;
```

Comment:
doesn't rzalloc already do this? same for the lines above

Reply:
Yep, a habit of mine. I'll add a comment and remove those.

Reply:
Done.